### DISCUSSION TITLE
Jiotv.be

### DISCUSSION THREAD

**Discussion by jitendrasd** [Jitendraunatti/JioTv#7] [General]
Bhai ye tera jiotv.be ekdum useless hai koi bhi channel 5min ke zyada nhi chalta. example ke liye koi bhi star ka channel le lo 5min ke baad fir refresh karna padta hai browse ko. kyuki jo link create hota hai usse block kar deta hai 5min ke baad. bhai tu pahle play kar ke dekh.

**Comment by Jitendraunatti** [comment]
OK SOURCE FAULT HAI . BUT KOI NA MAI DEKHTA HU  KUCH ALTERNATE
